src/components/task/index.cjsx

```diff
@@ -32,24 +32,31 @@ module.exports = React.createClass
   contextTypes:
     router: React.PropTypes.func
 
-  getInitialState: ->
+  setStepKey: ->
     {stepIndex} = @context.router.getCurrentParams()
     # url is 1 based so it matches the breadcrumb button numbers
     crumbKey = if stepIndex then parseInt(stepIndex) - 1 else @getDefaultCurrentStep()
+    @setState({currentStep: crumbKey})
+
+  getInitialState: ->
     {
-      currentStep: crumbKey
+      currentStep: 0
```

Comment:
Does this still work with the student dashboard sending the student directly to a specific step?